avaframe/out1Peak/outPlotAllPeak.py

```diff
@@ -110,15 +112,28 @@ def plotAllPeakFields(avaDir, cfgFLAGS, modName, demData=''):
             rowsMaxPlot = (rowsMax+1)*cellSize
             colsMinPlot = colsMin*cellSize
             colsMaxPlot = (colsMax+1)*cellSize
-            im0 = ax.imshow(demConstrained, cmap='Greys', extent=[colsMinPlot, colsMaxPlot, rowsMinPlot, rowsMaxPlot], origin='lower', aspect='equal')
-            im1 = ax.imshow(data, cmap=cmap, norm=norm, extent=[colsMinPlot, colsMaxPlot, rowsMinPlot, rowsMaxPlot], origin='lower', aspect='equal')
+
+            # add DEM hillshade with contour lines
+            ls = LightSource(azdeg=pU.azimuthDegree, altdeg=pU.elevationDegree)
+            im0 = ax.imshow(ls.hillshade(demConstrained, vert_exag=pU.vertExag, dx=demConstrained.shape[1],
+                dy=demConstrained.shape[0]), cmap='gray', extent=[colsMinPlot, colsMaxPlot,
+                rowsMinPlot, rowsMaxPlot], origin='lower', aspect='equal', zorder=1)
+            X, Y = oP._setCoordinateGrid(colsMinPlot, rowsMinPlot, cellSize, demConstrained)
+            CS =  ax.contour(X, Y, demConstrained, colors=['gray'], levels=pU.hillshadeContLevs, alpha=1.,
+                linewidths=0.5, zorder=2)
+            ax.clabel(CS, CS.levels, inline=True, fontsize=8,zorder=3)
+            pU.putInfoBox(ax, '- elevation [m]', location='upperLeft', color='gray',
+                hAlignment='left', alphaF=1.0)
+
```

Comment:
could we wrap this into a function so that we can use it for other plots too?

Reply:
made a function for this